sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -1732,9 +1732,9 @@ case class Left(str: Expression, len: Expression, child: Expression) extends Run
     Examples:
       > SELECT _FUNC_('Spark SQL ');
        10
-      > SELECT CHAR_LENGTH('Spark SQL ');
+      > SELECT _FUNC_('Spark SQL ');
```

Comment:
Would it be better show examples for binary data instead of demonstrating synonyms of the function name?

Reply:
And it would be nice to show to users all synonyms in a separate list somewhere in the description

Reply:
Yes. +1

Reply:
https://issues.apache.org/jira/browse/SPARK-29241